app/src/api/learningJourneyCourse.js

```diff
@@ -0,0 +1,29 @@
+import axios from "axios";
+import { LEARNING_JOURNEY_COURSE_ENDPOINT } from "./config";
+
+const axiosLJInstance = axios.create({
+    baseURL: LEARNING_JOURNEY_COURSE_ENDPOINT,
+    timeout: 5000,
+    headers: { "X-Custom-Header": "foobar" },
+});
+
+export const getLearningJourneyCoursesById = async (LJId) => {
+    try {
+        const res = await axiosLJInstance.get(`/${LJId}`);
+        if (res) {
+            return extractCourseIdsFromLJAndCourseIdsObjects(res.data);
```

Comment:
Hmm yea I agree with you. I think might be a bit too verbose. You can decide if you want to change or leave it in and merge.